.circleci/config.yml

```diff
@@ -56,4 +56,7 @@ jobs:
 
       - run:
           name: Run Integration Tests
-          command: docker run --link api:api --link dashboard:dashboard -e CYPRESS_BASE_URL=http://dashboard:3000 -e CYPRESS_APIROOT=http://api:5001 --rm -v /home/circleci/.cache:/root/.cache -v $(pwd):/home --workdir /home cypress/base:8 yarn cypress-ci
\ No newline at end of file
+          command: docker run --link api:api --link dashboard:dashboard -e CYPRESS_BASE_URL=http://dashboard:3000 -e CYPRESS_APIROOT=http://api:5001 --rm -v /home/circleci/.cache:/root/.cache -v $(pwd):/home --workdir /home cypress/base:8 yarn cypress-ci
+
+      - store_artifacts:
```

Comment:
Is this intentional that we now want to store screenshots for our CI runs of Cypress tests? Is this configuration now being used?

Reply:
I added it when I was debugging.  Don't think it hurts anything?

Reply:
The only thing I was wondering is whether there would be a storage issue with writing screenshots to a directory on every CI run. But I suppose that's probably not how this works. 

In general, any CI failure should be reproducible locally, so I'm not sure what adding this gets for us.

Reply:
I don't care that much, I can remove it.